app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Initial placements findings
+description: Initial findings on placements show they are more complex than we initially thought.
+
+date: 2021-05-01
+---
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{% from "figure/macro.njk" import appFigure with context %}
+
+We’ve been continuing our work around [collecting placement details](/register-trainee-teachers/collecting-placement-details/) through exploratory research and usability testing on the prototype.
+
+Dates of research:
+ 
+* round 5: 4 -12 March 2021, 7 participants, all SCITT
+* round 6: 29 March - 12 April 2021, 9 participants, 4 HEI, 5 SCITT, (8 EY Employment, 3 EY Graduate, 2 EY UG, 1SD Salaried, 2 SD tuition fee)
+
+
+## What we learned
+
+All the users we tested with understood what they were being asked to do in regards to providing placement details.
+
+Most of the people we spoke to could provide information for the first placement when registering a trainee in September, but for some this meant changing their current work practices. 
+
+> “We’d struggle with this, wouldn’t we? We’d have to go and get this information from our placement team.” 
+> Round 6 P7 (Scitt)
+
+> “If I’m doing it in August I’ll do “No, I’m doing it later”. If there are any changes I’d have to go back in and amend [them]. If I was keying in September I would put the first. I wouldn’t put the second as there might be changes in the Autumn. [I prefer] keying in the correct information rather than going in and changing it.” 
+> Round 5 P3 (HEI, EY employment)
+
+The rationale behind asking for placements is opaque and can lead to additional frustrations.
+
+> “Compliance states that we need two, but why does the DfE need to know where the person is placed? They just need to know that they are placed?” 
+> Round 6 P6 (SCITT)
+
+We found that further placements are often unknown at the point of requesting TRN.
+
+All users feel confident that Register will enable them to change or finalise the placement info at a later point.
+
+> “I've always keyed in the number of days. I’ve never not. We have our year planned out so to me it doesn't make any difference as long as we can change. I’ve had to change things due to the pandemic.” 
+> Round 5 P3 (SCITT)
+
+> “We had a couple of times when a trainee changed placements, I wonder how I would record that? I think I would just add another placement I’m guessing and change the previous placement to just 2 weeks?”
+> Round 5 P6 (SCITT)
+
+We learned that providers’ approach to placements was more varied than we anticipated. We assumed that a trainee would have two linear placements and that terminology would be consistent across the ITT market. Neither turned out to be true and our initial designs did not fit with user needs.
```

Comment:
I wonder if there's a more plain english way of referring to `ITT market`

Reply:
...... and that ITT provider terminology would be consistent.

Reply:
Agreed. Gregs recommendation added.